### ISSUE TITLE
Can I revert to the version without support for variable speed pumps?

### ISSUE THREAD

**Issue by nickcurmi** [dustindclark/homebridge-pentair-intellicenter#144]
**Describe Your Problem:**
I updated to the latest version, but I’m having an issue with the new VSP feature. I have multiple automations that call features on the intellicenter that have flow rate pre determined. For example, I have a scene that turns on my water feature. Now there are two sliders under water feature, one for an on/off (like was there before) but there is also a slider for the speed of the pump. After install, but scenes turn the speed slider from its preset amount (56%) to zero and then back again. However, it never turns the on/off slider to the on or off position as I want it to do. The result is the water feature never actually turns on/off.

while I imagine that quite a few individuals have a need to support variable speed, I actually don’t have a need for it. Wondering if I can get access to the prior version and install that.

**Logs:**

```
Show the Homebridge logs here, remove any sensitive information.
```

**Plugin Config:**

```json
Show your Homebridge config.json here, remove any sensitive information.
```

**Screenshots:**

**Environment:**

* **Plugin Version**:
* **Homebridge Version**: 
* **Node.js Version**: 
* **NPM Version**: 
* **Operating System**:

**Comment by dustindclark** [comment]
Yes, you can downgrade on the Homebridge console. I'll see if i can recreate.DustinSent from my iPhoneOn Apr 15, 2023, at 3:20 AM, nickcurmi ***@***.***> wrote:﻿

Describe Your Problem:
I updated to the latest version, but I’m having an issue with the new VSP feature. I have multiple automations that call features on the intellicenter that have flow rate pre determined. For example, I have a scene that turns on my water feature. Now there are two sliders under water feature, one for an on/off (like was there before) but there is also a slider for the speed of the pump. After install, but scenes turn the speed slider from its preset amount (56%) to zero and then back again. However, it never turns the on/off slider to the on or off position as I want it to do. The result is the water feature never actually turns on/off.
while I imagine that quite a few individuals have a need to support variable speed, I actually don’t have a need for it. Wondering if I can get access to the prior version and install that.
Logs:
Show the Homebridge logs here, remove any sensitive information.

Plugin Config:
Show your Homebridge config.json here, remove any sensitive information.
Screenshots:

Environment:

Plugin Version:
Homebridge Version: 
Node.js Version: 
NPM Version: 
Operating System:

**Comment by dustindclark** [comment]
Can you attach screenshots of your scene configuration?DustinSent from my iPhoneOn Apr 15, 2023, at 3:20 AM, nickcurmi ***@***.***> wrote:﻿

Describe Your Problem:
I updated to the latest version, but I’m having an issue with the new VSP feature. I have multiple automations that call features on the intellicenter that have flow rate pre determined. For example, I have a scene that turns on my water feature. Now there are two sliders under water feature, one for an on/off (like was there before) but there is also a slider for the speed of the pump. After install, but scenes turn the speed slider from its preset amount (56%) to zero and then back again. However, it never turns the on/off slider to the on or off position as I want it to do. The result is the water feature never actually turns on/off.
while I imagine that quite a few individuals have a need to support variable speed, I actually don’t have a need for it. Wondering if I can get access to the prior version and install that.
Logs:
Show the Homebridge logs here, remove any sensitive information.

Plugin Config:
Show your Homebridge config.json here, remove any sensitive information.
Screenshots:

Environment:

Plugin Version:
Homebridge Version: 
Node.js Version: 
NPM Version: 
Operating System:

**Comment by astrostl** [comment]
I was experiencing the same issue. As this repository no longer seems to be maintained I created https://github.com/astrostl/homebridge-pentair-intellicenter-ai which brought in a fork from another repo that has a (default off) option to disable the fan feature. If interested (caveat emptor!) you can install it by searching Homebridge for the 'homebridge-pentair-intellicenter-ai' plugin that I hope to get verified quickly.

**Comment by dustindclark** [comment]
This repo is still maintained (albeit, in my very limited free time). Was a PR submitted for this? Forking doesn't make sense.

Deleting spam comments that were bulk added to all issues. @astrostl I'd be happy to review a PR.

**Comment by astrostl** [comment]
I submitted PRs to https://github.com/Windscar/homebridge-pentair-intellicenter — an existing fork — before proceeding with my own after days without a response. As I imagine they did, I took this repo to be inactive as it hasn't been updated for years, has outstanding dependabot PRs from 2023, and unresolved/unresponded issues. Which isn't nearly a complaint as we all have lives and priorities and open source maintenance can be a slog. I'm grateful for what you've already done.

I wouldn't characterize the comments I made as spam, they weren't bulk added, and they weren't added to all issues — just those that have been apparently addressed after examining the issue and customizing the response. I apologize for any extent to which they came across as "competitive", which wasn't my intent — just to fix bugs, add features, and make those requesting them aware that it had happened.

So far I've changed thousands of LOC (!!) and counting: https://github.com/astrostl/homebridge-pentair-intellicenter-ai/compare/dustindclark...master . That's because I'm using Claude Code (genAI). The three most straightforward paths I see: we stay AI and non-AI forked OR we pick one repo and share commit access OR we try to come up with some sort of review arrangement that we both find agreeable.
